src/state/routing/slice.ts

```diff
@@ -0,0 +1,117 @@
+import { BaseProvider, JsonRpcProvider } from '@ethersproject/providers'
+import { createApi, fetchBaseQuery, FetchBaseQueryError } from '@reduxjs/toolkit/query/react'
+import { Protocol } from '@uniswap/router-sdk'
+// eslint-disable-next-line no-restricted-imports
```

Comment:
I don't think this file is lazy-loaded. It's not `await import`ed anywhere. AFAIK importing `ChainId` will import the entire `@uniswap/smart-order-router` with the initial package.

However, I think if you import just the _type_, then it will be lazy-loaded, as the type is only used during build

    import type { ChainId } from '@uniswap/smart-order-router'